docs/openshift/debugging_guide.md

```diff
@@ -1,70 +1,86 @@
 # Openshift 4.x CI/CD debugging guide
+
 Some tidbits around debugging CI/CD issues on Openshift and info for interacting with Openshift
 
 # Github actions
+
 Github has excellent [docs](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) for learning actions. Give this a read to understand the high level view of actions.
 
 # Openshift CLI connections
+
 - Once your changes are merged, the way that changes move to openshift occur in 2 ways: CLI calls, and pushing artifacts (docker images)
 - The openshift CLI is very robust with great docs, you can drill down into the help of any part like so: `oc -h` for overview, then `oc apply -h` for docs specifically around the `apply` command. These come with examples too!
 - Always check the Openshift cluster version, at the time of writing it is 4.10 - you can find docs for the majority of our workloads here: [4.10 API Ref](https://docs.openshift.com/container-platform/4.10/rest_api/workloads_apis/workloads-apis-index.html)
 - Additional reading resource: [BCGov Openshift Wiki](https://github.com/BCDevOps/openshift-wiki/tree/master/docs)
 
 # Openshift DeploymentConfigs
+
 The majority of Core's services are still using DeploymentConfigs and these follow a hiearchal pattern where child objects (not every object is a child) inherit configuration from the parent object, that hierchary could be described as follows:
 
 |-HorizontalPodAutoscaler
 |+DeploymentConfig
- |+ReplicationController
-  |+Pod(s)
-   |+Container(s)
+|+ReplicationController
```

Comment:
maybe ignore md files for now until we create new linting rules for them

Reply:
Changed my mind.
Installed a md preview extension (added to extension recommendations!) and neither of these look good.